readthedocs/vcs_support/backends/git.py

```diff
@@ -122,11 +122,15 @@ def validate_submodules(self, config):
                 submodules_include[path] = submodules[path]
             submodules = submodules_include
 
+        invalid_submodules = []
         for path, submodule in submodules.items():
             try:
                 validate_submodule_url(submodule.url)
             except ValidationError:
-                return False, []
+                invalid_submodules.append(path)
+
+        if invalid_submodules:
+            return False, invalid_submodules
```

Comment:
The docstring has to be updated to communicate that when it's False, invalid submodules are returned instead of valid ones.